Makefile

```diff
@@ -879,13 +889,14 @@ $(BUILD_DIR)/goddard.txt: $(BUILD_DIR)/sm64_prelim.elf
 	$(call print,Getting Goddard size...)
 	$(V)python3 tools/getGoddardSize.py $(BUILD_DIR)/sm64_prelim.map $(VERSION)
 
-$(BUILD_DIR)/asm/debug/map.o: asm/debug/map.s $(BUILD_DIR)/sm64_prelim.elf
-	$(call print,Assembling:,$<,$@)
-	$(V)python3 tools/mapPacker.py $(BUILD_DIR)/sm64_prelim.elf $(BUILD_DIR)/bin/addr.bin $(BUILD_DIR)/bin/name.bin
-	$(V)$(CROSS)gcc -c $(ASMFLAGS) $(foreach i,$(INCLUDE_DIRS),-Wa,-I$(i)) -x assembler-with-cpp -MMD -MF $(BUILD_DIR)/$*.d  -o $@ $<
+$(BUILD_DIR)/sm64.sym.o: $(BUILD_DIR)/sm64_prelim.elf
+	$(call print,Generating symbol table:,$(@F))
+	$(V)tools/n64sym $(DEBUG_EXPORT_ALL_LINES_FLAG) --cross $(CROSS) $(BUILD_DIR)/sm64_prelim.elf
```

Comment:
Why introduce a whole separate tool when `nm` exists?

nm './build/us_n64/sm64.elf'
801617b8 T able_to_grab_object
801891d8 T abs_angle_diff
80168cdc T act_air_hit_wall
8016a3ac T act_air_throw
80179ff0 T act_air_throw_land
8016c1cc T act_backflip
80177504 T act_backflip_land
80177004 t act_backflip_land.part.0
8017a340 T act_backflip_land_stop
8016ca10 T act_backward_air_kb

Reply:
the old implementation used `nm -S`, but this tool has more resolution, including line numbers at call sites (optionally every single line), and the interface code uses dma instead of being loaded in memory, so even though it's ~300k of data now (1MB if every line is included) instead of 150k, it's only a ROM impact (or purely an SD card impact once #752 is merged)